browser/src/shared/code-hosts/shared/hoverAlerts.tsx

```diff
@@ -1,27 +1,27 @@
 import { Observable, of } from 'rxjs'
 import { catchError, map, startWith, switchMap } from 'rxjs/operators'
-import { HoverAlert } from '../../../../../shared/src/hover/HoverOverlay'
+import { HoverAlert } from 'sourcegraph'
```

Comment:
To make sure no runtime types are imported (which doesn't work)
```suggestion
import type { HoverAlert } from 'sourcegraph'
```